src/libraries/System.Composition.TypedParts/src/PACKAGE.md

```diff
@@ -0,0 +1,87 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+`System.Composition.TypedParts` is part of the Managed Extensibility Framework (MEF) 2.0, a composition library for .NET that enables dependency injection through attributes.
+
+Provides `ContainerConfiguration` and some extension methods for the Managed Extensibility Framework (MEF).by MEF.
+
+## Key Features
+
+<!-- The key features of this package -->
+
+* Provides container configuration.
+
+## How to Use
+
+<!-- A compelling example on how to use this package with code, as well as any specific guidelines for when to use the package -->
+
+Register parts from an entire assembly.
+
+```csharp
+using System.Composition;
+using System.Composition.Hosting;
+using System.Reflection;
+
+// Register all parts from the current assembly
+var configuration = new ContainerConfiguration()
+    .WithAssembly(Assembly.GetExecutingAssembly());
+
+using CompositionHost container = configuration.CreateContainer();
+
+var handlers = container.GetExports<IHandler>();
+foreach (var handler in handlers)
+{
+    handler.Handle();
+}
+// HandlerA is handling.
+// HandlerB is handling.
+
+public interface IHandler
+{
+    void Handle();
+}
+
+[Export(typeof(IHandler))]
+public class HandlerA : IHandler
+{
+    public void Handle() => Console.WriteLine("HandlerA is handling.");
+}
+
+[Export(typeof(IHandler))]
+public class HandlerB : IHandler
+{
+    public void Handle() => Console.WriteLine("HandlerB is handling.");
+}
+```
+
+## Main Types
+
+<!-- The main types provided in this library -->
+
+The main types provided by this library are:
+
+* `System.Composition.Hosting.ContainerConfiguration`
+* `System.Composition.CompositionContextExtensions`
+
+## Additional Documentation
+
+<!-- Links to further documentation. Remove conceptual documentation if not available for the library. -->
+
+* [Managed Extensibility Framework (MEF)](https://learn.microsoft.com/dotnet/framework/mef/)
+
+## Related Packages
+
+<!-- The related packages associated with this package -->
+
```

Comment:
For all of these, should we link to the `System.Composition` meta-package?

Reply:
Sure